### ISSUE TITLE
Is there a way to try CodeGeeX in an online web app?

### ISSUE THREAD

**Issue by ndattani** [THUDM/CodeGeeX#67]
I'm hesitant to download and install new software these days, since almost everything is available via online web apps. For example I no longer need my own TeX distribution since Overleaf has it all. I no longer have Microsoft Word because I can use Google Dogs with my web browser.

Is it possible to try CodeGeeX this way?

**Comment by Stanislas0** [comment]
> I'm hesitant to download and install new software these days, since almost everything is available via online web apps. For example I no longer need my own TeX distribution since Overleaf has it all. I no longer have Microsoft Word because I can use Google Dogs with my web browser.
> 
> Is it possible to try CodeGeeX this way?

Sure! Please try CodeGeeX directly in Cloud Studio (a powerful cloud IDE), by clicking [this link](https://cloudstudio.net/templates/h0kvkZvoO0U). You can use CodeGeeX in your browser without downloading.

**Comment by ndattani** [comment]
@Stanislas0 that appears to require me to sign-in using either WeChat or GitHub or [coding.net](https://coding.net/). I was wondering if there's something like [Octave online](https://octave-online.net/) which allows me to quickly use Octave in a web browser without signing in.

**Comment by Stanislas0** [comment]
> @Stanislas0 that appears to require me to sign-in using either WeChat or GitHub or [coding.net](https://coding.net/). I was wondering if there's something like [Octave online](https://octave-online.net/) which allows me to quickly use Octave in a web browser without signing in.

Yeah, I understand. Why not have a try on our online [DEMO](http://codegeex.cn/playground)? This doesn't require any sign-in, but it just supports simple functions. If you want more features of CodeGeeX, I suggest using IDE extensions for a better experience.

**Comment by ndattani** [comment]
@Stanislas0  thanks! I didn't like that the "insecure" icon showed up next to the URL, in my browser, but I tried it anyway. Why is it that I can only generate one line of code at a time? Isn't that slow and tedious?

**Comment by andiwang1993** [comment]
Hi, you can try it on Gitpod which only needs to login with github or gitlab or bitbucket;) Search codegeex in the market and you will find it, happy coding!

**Comment by ndattani** [comment]
@andiwang1993 I've created a GitPod account now. What do you mean by the "market"? I don't see how to use CodeGeeX with GitPod :'(

**Comment by andiwang1993** [comment]
> 

After login, you can go to https://gitpod.io/workspaces/ to find if there are some workspaces, if not, create one and then open this workspace, click "Extensions" button the left sidebar([https://imgur.com/N7WSSPj](https://imgur.com/N7WSSPj)), and search our extension in the marketplace and install it, after this you can start to use our extension ;)
